index.js

```diff
@@ -8,6 +8,7 @@ const nycCommands = {
 	instrument: [null, 'instrument'],
 	checkCoverage: [null, 'report', 'check-coverage'],
 	report: [null, 'report'],
+	cache: [null, 'instrument'],
```

Comment:
This is not needed as we can just use `nycCommands.instrument` (see suggested code changes below).

Reply:
I've pushed up the recommended changes, but I'm still a little confused as to the purpose of the `nycCommands` block at the top of the schema file.  What does it do?

Reply:
It serves the same purpose as code such as `const MAGIC_VALUE = 53;`.  Instead of repeating `[null, 'instrument']` for each option we reference `nycCommands.instrument`.  This module was created without much feedback so if you have suggestions for how to improve the code readability I'm :+1:, but any such changes would need to be a separate PR.